lib/ace/ext/katex-demo.js

```diff
@@ -0,0 +1,26 @@
+define(function(require, exports, module) {
```

Comment:
any sense to start using CoffeeScript and jQuery? 

Also, let's move this script into ` lib/ace/ext/papeeria/`, just to keep our stuff in a single place.